scripts/env-vars-json.xsh

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env python3
+"""
+Script to save environment variables to a JSON file.
+Takes environment variable names as command line arguments and saves their values to JSON.
```

Comment:
I do not like this approach. Seems complicated to maintain and adds a higher risk of leaking the secrets.

I think this should be done on the extension side instead.